.github/workflows/tests.yml

```diff
@@ -16,27 +16,22 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
 jobs:
   unit-tests:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
-        release: ['', 'branched_fedora']
+        release: ["", "fedora-43"]
```

Comment:
Just a nitpick but I think we can have this use '' instead of "" as that's the style of the file everywhere in the file. :)

Reply:
I dont think i can influence that :( 

This comes from:
+        release: [{$ matrix_data | map(attribute='release') | map('tojson') | join(', ') $}]